extension/awsxrayproxy/extension.go

```diff
@@ -19,29 +19,39 @@ import (
 
 	"go.opentelemetry.io/collector/component"
 	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/proxy"
 )
 
 type xrayProxy struct {
 	logger *zap.Logger
 	config *Config
+	server proxy.Server
 }
 
 var _ component.Extension = (*xrayProxy)(nil)
 
 func (x xrayProxy) Start(ctx context.Context, host component.Host) error {
-	// TODO(anuraaga): Implement me
+	go x.server.ListenAndServe()
+	x.logger.Info("X-Ray proxy server started on " + x.config.ProxyConfig.Endpoint)
 	return nil
 }
 
 func (x xrayProxy) Shutdown(ctx context.Context) error {
-	// TODO(anuraaga): Implement me
-	return nil
+	return x.server.Close()
```

Comment:
Can we ensure this honors a timeout if one is set on `ctx`?

Reply:
Switched to shutting down with context